src/parser.ts

```diff
@@ -627,7 +637,30 @@ export const parseOpcodeIns = (str: string, pc: number): BpfInstructionPair => {
   return { ins: undefined, rest: str };
 };
 
-export const parseLine = (rawLine: string, idx: number): ParsedLine => {
+const parseCSourceLine = (str: string, idx: number): ParsedLine | null => {
+  const { match } = consumeRegex(RE_C_SOURCE_LINE, str);
+  if (match) {
```

Comment:
super nit. I like an early exit to prevent nesting
```suggestion
  if (!match) {
    return null;
```